### PR TITLE
Add Class Names

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**
Adds a bunch of class names all over the place to make post-compile styling easier.

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

**Review by daniel-heppner-ibigroup — CHANGES_REQUESTED** [review]
This looks pretty good but I have a few thoughts about redundant things. Also I'd like to be able to test it with the new ItinerarySupplement component

File: lib/components/app/app.css [hunk 1]
```diff
@@ -33,12 +33,14 @@
...
 /* Don't show route viewer link in the app menu on desktop as it is in the navbar */
 .app-menu .app-menu-route-viewer-link,
+.app-menu .app-menu-nearby-viewer-link,
 .app-menu .app-menu-trip-planner-link,
 .app-menu #app-menu-locale-selector {
   display: none;
 }
 @media (max-width: 768px) {
   .app-menu .app-menu-route-viewer-link,
+  .app-menu .app-menu-nearby-viewer-link,
```
[the remark this answers is not in the document]
Comment (miles-grant-ibigroup) [reply lib/components/app/app.css:43]: I think there is a user who wants to style these separately. I also tried to keep changes as minimal as possible

File: lib/components/narrative/metro/attribute-utils.tsx [hunk 2]
```diff
@@ -3,11 +3,12 @@ import { Itinerary, Leg } from '@opentripplanner/types'
 import coreUtils from '@opentripplanner/core-utils'
 import React from 'react'
 
+export const getFirstTransitLeg = (itinerary: Itinerary): Leg | undefined =>
+  itinerary.legs?.find((leg: Leg) => leg?.from?.vertexType === 'TRANSIT')
+
 export const getFirstTransitLegStop = (
```
[the remark this answers is not in the document]
Comment (miles-grant-ibigroup) [reply lib/components/narrative/metro/attribute-utils.tsx:9]: It's used in the metro itinerary!

[the remark this answers is not in the document]
Comment (daniel-heppner-ibigroup) [reply lib/components/narrative/metro/attribute-utils.tsx:9]: If it wasn't clear this comment is about `getFirstTransitLegStop`. Literally all it does is return the stop from an itinerary. I don't know that we really need a whole function to do that. There's no logic there

[the remark this answers is not in the document]
Comment (miles-grant-ibigroup) [reply lib/components/narrative/metro/attribute-utils.tsx:9]: It's shorter to write out the method than to copy paste that line everywhere!

**Review by daniel-heppner-ibigroup — APPROVED** [review]

**Review by amy-corson-ibigroup — APPROVED** [review]
I think this makes sense! Beautiful class names

**Pull request merged**